AgoraSocialApp/Models/ApplicationUser.cs

```diff
@@ -6,10 +6,12 @@
 
 namespace AgoraSocialApp.Models
 {
-    //public class ApplicationUser : Raven.Identity.IdentityUser
-    public class ApplicationUser : IdentityUser
+    public class ApplicationUser : Raven.Identity.IdentityUser
     {
-        //public string FullName { get; set; }
+        public string FirstName { get; set; }
```

Comment:
are you adding handle here or to separate model?

Reply:
You're right, handle should be here